src/openvic-simulation/types/Sliders.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "openvic-simulation/types/fixed_point/FixedPoint.hpp"
+#include "openvic-simulation/utility/Getters.hpp"
+
+namespace OpenVic {
+  // TODO: make struct for tariff slider, unlike other sliders, you can have negative values
+  struct slider_value_t {
```

Comment:
`slider_value_t` feels wrong to me. I think `RestrictedValue` might be a better name.
This type is not directly tied to a UI slider element. It merely restricts tax and tariff rates between min and max.

Reply:
Types are named PascalCase without pre- or suffix either way.

Reply:
This is a more complex object. It has dynamic state validation.

Reply:
I see, I will change it to `SliderValue`